src/components/grid-view/GridView.js

```diff
@@ -112,6 +98,7 @@ class GridView extends React.Component<Props> {
                 width={width}
                 gridClassName="bdl-GridView-body"
                 rowClassName="bdl-GridView-tableRow"
+                scrollToIndex={0}
```

Comment:
It'd be an interesting enhancement in the future if we could maybe cache this value when they've scrolled down the gridview so that when they leave that folder and come back, they're at the same position in the grid

Reply:
I think so too. It seems like ListView also always returns to the top of the list, so the feature could be added there as well.